src/postprocess.ts

```diff
@@ -3,55 +3,57 @@
  * https://github.com/facebook/jest/blob/9b157c3a7c325c3971b2aabbe4c8ab4ce0b0c56d/packages/babel-jest/src/index.js
  */
 import * as jestPreset from 'babel-preset-jest';
-import { JestConfig, PostProcessHook, PostProcessorOptions, TransformOptions } from './jest-types';
+import {JestConfig, PostProcessHook, PostProcessorOptions, TransformOptions} from './jest-types';
 import * as babel from 'babel-core';
-import { CompilerOptions } from 'typescript/lib/typescript';
+import {CompilerOptions} from 'typescript/lib/typescript';
 
 function createBabelTransformer(options: PostProcessorOptions) {
-	options = {
-		...options,
-		plugins: (options && options.plugins) || [],
-		presets: ((options && options.presets) || []).concat([jestPreset]),
-		retainLines: true,
-	};
-	delete options.cacheDirectory;
-	delete options.filename;
+    options = {
+        ...options,
+        plugins: (options && options.plugins) || [],
+        presets: ((options && options.presets) || []).concat([jestPreset]),
+        retainLines: true,
```

Comment:
I see that babel does have an inputSourceMap though. What problem are we trying to solve exactly?